modules/node-shared/src/main/scala/org/tessellation/node/shared/domain/seedlist/Alias.scala

```diff
@@ -0,0 +1,18 @@
+package org.tessellation.node.shared.domain.seedlist
+
+import cats.syntax.all._
+
+import derevo.cats.eqv
+import derevo.derive
+import fs2.data.csv.CellDecoder
+
+@derive(eqv)
+case class Alias(value: String)
+
+object Alias {
+
+  implicit val maybeAliasCellDecoder: CellDecoder[Option[Alias]] = CellDecoder.stringDecoder.map { s =>
+    if ("".equals(s)) none else Alias(s).some
```

Comment:
Is `" "` a correct alias (just the spaces)? If not then I would use `s.trim.nonEmpty`. Also you can get rid of explicit `none` by using `Option.when` or `Option.unless`:

Alternatively, you can get rid of explicit `none`:

```scala
Option.when(s.trim.nonEmpty)(Alias(s))
```

Reply:
Changed to `Option.when(s.trim.nonEmpty)(Alias(s))`